lib/OpenQA/Utils.pm

```diff
@@ -899,4 +900,6 @@ sub find_video_files { path(shift)->list_tree->grep(VIDEO_FILE_NAME_REGEX) }
 # workaround https://github.com/mojolicious/mojo/issues/1629
 sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/ }
 
+sub is_url { return !!Mojo::URL->new(shift)->scheme }
```

Comment:
In consistency with `looks_like_number` from `Scalar::Utils`:

```suggestion
sub looks_like_url_with_scheme { Mojo::URL->new(shift)->scheme ne '' }
```

I guess `!!` vs. `ne ''` makes no real difference in practice but I wouldn't exclude other falsy values like `0` here. (Scheme is not supposed to return `undef`.)

Reply:
The function name is better.

Reply:
Scheme returns `undef`, see the log
```
Use of uninitialized value in string ne at /home/liuxiaojing/code/repos/openQA/lib/OpenQA/Utils.pm line 903.
```

Reply:
Ah, then you can keep your `!!` version.